composer.json

```
@@ -70,6 +70,7 @@
         "drupal/environment_indicator": "4.0.10",
         "drupal/exclude_node_title": "1.4.0",
         "drupal/externalauth": "2.0.3",
+        "drupal/extlink": "^1.7",
```

```suggestion
        "drupal/extlink": "1.7",
```
We typically pin to a specific release in order to try to keep or builds more deterministic.